src/VisualStudio/RemoteHostClientMock/Remote/RemoteHostClientFactory.cs

```diff
@@ -16,20 +16,13 @@ internal class RemoteHostClientFactory : IRemoteHostClientFactory
     {
         public async Task<RemoteHostClient> CreateAsync(Workspace workspace, CancellationToken cancellationToken)
         {
-            try
+            // this is the point where we can create different kind of remote host client in future (cloud or etc)
```

Comment:
same this belong to https://github.com/dotnet/roslyn/pull/16226

once that PR checked in, this change should go away